tests/functional/src/test_client.py

```diff
@@ -1,14 +1,15 @@
 class TestSimple:
```

Comment:
Need to add `screenshot.py`, `sessions.py`, `document.py` with tests for every implemented method.